metafacture-json/src/main/java/org/metafacture/json/JsonDecoder.java

```diff
@@ -34,6 +34,7 @@
 import java.util.Arrays;
 import java.util.List;
 import java.util.stream.Collectors;
```

Comment:
This import is unused now.

Reply:
Thanks, fixed. We need those (annoying) checks from metafacture-fix here :-)

Reply:
My thoughts exactly! :)